bids/modeling/tests/test_transformations.py

```diff
@@ -58,14 +58,14 @@ def sparse_run_variable_with_missing_values():
 
 def test_convolve(collection):
     rt = collection.variables['RT']
-    transform.Convolve(collection, 'RT', output='reaction_time')
+    transform.Convolve(collection, ['RT'], output=['reaction_time'])
     rt_conv = collection.variables['reaction_time']
 
     assert rt_conv.values.shape[0] == \
         rt.get_duration() * collection.sampling_rate
 
-    transform.ToDense(collection, 'RT', output='rt_dense')
-    transform.Convolve(collection, 'rt_dense', output='dense_convolved')
+    transform.ToDense(collection, ['RT'], output=['rt_dense'])
+    transform.Convolve(collection, 'rt_dense', derivative=True)
```

Comment:
Should we add a test that explicitly tests the names of the 3 new variables?

What would the be in this instance?

Reply:
Ping @jdkent

Reply:
I can add the explicit tests for names.